_pages/en/technical-platform.md

```diff
@@ -86,7 +86,15 @@ More detailed Azure DevOps Pipeline doc: [Azure Pipeline](../assets/docs/azure-p
 
 ## Security Implications
 
-- Things we have done to make it more secure.
+- Using GraphQL mobible friendly API with strong input validation to prevent hacking the database.
+- Using containers to minimize attack surface--less code bug, less chance to be attacked. 
+- Continuous deployement let us fix security problems quickly- pipleline has "npm install",  will check all the dependence security issue. 
+- Scrambling personal information in both protypes before saving to database.
+- Only the delivery team members can access to the data which gathered from protypes testing. 
+- Only the people with permission can access to RCMP Azure.
```

Comment:
May need to re-phrase this and change RCMP azure to Azure? Or say that access to infrastructure is based on infrastructure's rules (running the code doesn't grant access to the infrastructure?) @sleepycat @keithiancolbourne ?

Reply:
I changed to "Only the people with permission can access to the project Azure."